### DISCUSSION TITLE
[Feature] Add Tag From Explore/Search Results

### DISCUSSION THREAD

**Discussion by rholloway** [immich-app/immich#14106] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When using the (web) UI, I would like the ability to add / remove tags in bulk from the search results and/or explore tabs. Right now, I can multi-select the photos in these views, but I can't apply a tag to them (although I can add them to an album)

The only way to apply tags to multi-select that I'm aware of is on the photos tab or from an album. The best location would be to be able to perform searches, and then multi-select the photos (either select all or pick and choose) and add tags from this page of results.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by TildenWinston** [comment]
This seems related to #14213 

Looks like the tag-asset import is missing from the search page, web/src/routes/(user)/search/[[photos=photos]]/[[assetId=id]]/+page.svelte, while it is present in the album page for example (web/src/routes/(user)/albums/[albumId=id]/[[photos=photos]]/[[assetId=id]]/+page.svelte)

The best work around seems to be to use the search to create a (temporary) album, tag the photos, and then delete the album

**Reply by Menghini** [reply]
Howdy, although it was close to #14213, it didn't solve OP's feature request. I believe I have covered it in PR #14368

**Comment by bo0tzz** [comment]
#14368
